src/libraries/System.Private.CoreLib/src/System/String.cs

```diff
@@ -28,17 +28,9 @@ public sealed partial class String : IComparable, IEnumerable, IConvertible, IEn
         /// <remarks>Keep in sync with AllocateString in gchelpers.cpp.</remarks>
         internal const int MaxLength = 0x3FFFFFDF;
 
-        // The Empty constant holds the empty string value. It is initialized by the EE during startup.
-        // It is treated as intrinsic by the JIT as so the static constructor would never run.
-        // Leaving it uninitialized would confuse debuggers.
-        //
-        // We need to call the String constructor so that the compiler doesn't mark this as a literal.
-        // Marking this as a literal would mean that it doesn't show up as a field which we can access
-        // from native.
-#pragma warning disable CS8618 // compiler sees this non-nullable static string as uninitialized
+        // The Empty constant holds the empty string value.
         [Intrinsic]
-        public static readonly string Empty;
-#pragma warning restore CS8618
+        public static readonly string Empty = "";
```

Comment:
We would have double initialization for this field. We should consider removing https://github.com/dotnet/runtime/blob/58b25dfa8fae5f5a382a6ffbe89e9aee479eb31c/src/mono/mono/metadata/appdomain.c#L180-L189

Reply:
Hmmm, I guess that removing this may have also impact on performance in some cases. I will leave it as it was.